awx/main/management/commands/inventory_import.py

```diff
@@ -131,7 +127,6 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('--inventory-name', dest='inventory_name', type=str, default=None, metavar='n', help='name of inventory to sync')
         parser.add_argument('--inventory-id', dest='inventory_id', type=int, default=None, metavar='i', help='id of inventory to sync')
-        parser.add_argument('--venv', dest='venv', type=str, default=None, help='absolute path to the AWX custom virtualenv to use')
```

Comment:
@wenottingham is removing a CLI option here fine with you?

Reply:
it doesn't do anything now, so yes.

Also, isn't there the other issue as to whether this command needs removed entirely for the OCP use case?

Reply:
yes, that OCP issue for the command is hard. I figure, I won't lose sleep over dropping a minor option when it needs major work in this other area.